src/main/java/com/mongodb/kafka/connect/source/schema/AvroSchema.java

```diff
@@ -223,6 +222,7 @@ static Schema createSchema(
         throw new IllegalStateException();
       case NULL:
       case ENUM:
+      case FIXED:
```

Comment:
I notices that we don't actually support the `fixed` Avro type, but this code that converts `org.apache.avro.Schema` to `org.apache.kafka.connect.data.Schema` was written as if we supported it.

Reply:
Created a separate PR: https://github.com/mongodb/mongo-kafka/pull/104.

> So there is a bug converting Fixed to bytes?

It's just that we don't support the `FIXED` type (`AvroSchema.validateJsonSchema` fails when when encounters one), but `AvroSchema.createSchema` was written in such a way that could mislead a reader into thinking that we support this type.